src/main/resources/assets/dreamcraft/lang/en_US.lang

```diff
@@ -946,8 +946,8 @@ tc.research_text.DISENCHANTINGTABLE=[DC] Oh, you can remove Enchants? Nice!
 tc.research_name.BREWINGSTAND=Brewing Stand
 tc.research_text.BREWINGSTAND=[MC] Oh, its more magical than a Cauldron!
-brewingstand.research_page.BREWINGSTAND.1=Brewing (or Alchemy) is the process of creating potions and splash potions by adding various ingredients to water bottles in a brewing stand. By placing bottles in the lower three slots of the brewing interface and an ingredient in the upper slot, you can distill the ingredient into the bottles and brew potions which may be consumed to grant an effect to the player. Every potion starts with a water bottle, made by filling a glass bottle at a water source or filled cauldron. The next step is to add a primary ingredient to create a base potion, which is usually Nether Wart, which creates an awkward potion and has no effects.
-brewingstand.research_page.BREWINGSTAND.2=By brewing a second ingredient into a base potion in the same manner, you can create a potion with a working effect. A third ingredient may be added to make the effect more intense or last longer, or turn the effect harmful. Finally, gunpowder can be added to a potion at any stage to convert it to a splash potion, which can be thrown (or fired using a dispenser) to disperse its effect in a radius. Each brewing step takes 20 seconds. Brewing can create very useful and lifesaving items. A good number of them are useful in combat by aiding the player or weakening enemies while others can save the players life if used quickly, like Fire Resistance or Healing.
+Minecraft.research_page.BREWINGSTAND.1=Brewing (or Alchemy) is the process of creating potions and splash potions by adding various ingredients to water bottles in a brewing stand. By placing bottles in the lower three slots of the brewing interface and an ingredient in the upper slot, you can distill the ingredient into the bottles and brew potions which may be consumed to grant an effect to the player. Every potion starts with a water bottle, made by filling a glass bottle at a water source or filled cauldron. The next step is to add a primary ingredient to create a base potion, which is usually Nether Wart, which creates an awkward potion and has no effects.
+Minecraft.research_page.BREWINGSTAND.2=By brewing a second ingredient into a base potion in the same manner, you can create a potion with a working effect. A third ingredient may be added to make the effect more intense or last longer, or turn the effect harmful. Finally, gunpowder can be added to a potion at any stage to convert it to a splash potion, which can be thrown (or fired using a dispenser) to disperse its effect in a radius. Each brewing step takes 20 seconds. Brewing can create very useful and lifesaving items. A good number of them are useful in combat by aiding the player or weakening enemies while others can save the players life if used quickly, like Fire Resistance or Healing.
```

Comment:
That was correct looking at the code
https://github.com/GTNewHorizons/NewHorizonsCoreMod/blob/bfb72d533633d131751781772f8fd189410845ae/src/main/java/com/dreammaster/scripts/ScriptTCCoreMod.java#L453-L454

Reply:
but when translated from the original it does not translate into Russian

Reply:
but why then is it standing here? Minecraft.research_page.BREWINGSTAND.3

Reply:
> but why then is it standing here? Minecraft.research_page.BREWINGSTAND.3

I don't know, thats how it was in the old scripts.
There are curently 3 keys that the code uses:
brewingstand.research_page.BREWINGSTAND.1
brewingstand.research_page.BREWINGSTAND.2
Minecraft.research_page.BREWINGSTAND.3

We can either leave it as that or fix it in the code and all lang files.